packages/adapters/src/adapters/Cornerstone3D/MeasurementReport.ts

```diff
@@ -392,6 +393,16 @@ export default class MeasurementReport {
                 const TrackingIdentifierValue =
                     TrackingIdentifierGroup.TextValue;
 
+                const TrackingUniqueIdentifierGroup =
+                    measurementGroupContentSequence.find(
+                        contentItem =>
+                            contentItem.ConceptNameCodeSequence.CodeMeaning ===
+                            TRACKING_UNIQUE_IDENTIFIER
+                    );
+
+                const TrackingUniqueIdentifierValue =
```

Comment:
Make sure this is a conditional getter - there is no guarantee you will find the tracking group id.
Also, could you update the top level description with where that tracking unique identifier gets written in the adapters code or in dcmjs?  Just makes it hard to figure out what is going on without that.